streamly-process.cabal

```diff
@@ -16,11 +16,9 @@ copyright:           Composewell Technologies
 category:            Streamly, Streaming, System
 stability:           Experimental
 tested-with:         GHC==9.4.2
-                   , GHC==9.2.1
+                   , GHC==9.2.2
```

Comment:
This should match the tested versions in the CI

Reply:
Yes, CI is for 9.2.2.